en/docs/reference/connectors/salesforce-marketing-cloud-connector/salesforce-marketing-cloud-email-prerequisites.md

```diff
@@ -0,0 +1,99 @@
+# Prerequisites for Sending an Email
+
+**Note**: Make sure authentication is set up before sending emails to prevent them from being bounced. Specifically, if the sending domain fails any of the SPF, DKIM, or DMARC checks, the email will be rejected.
+
+To ensure reliable email delivery and avoid bounces, it's important to properly configure the following authentication mechanisms:
+
+1. Sender Policy Framework (SPF): SPF validates that Salesforce is authorized to send emails on your domain’s behalf. [Learn more about SPF](https://help.salesforce.com/s/articleView?id=emailadmin_spf_overview.htm&type=5).
+2. DomainKeys Identified Mail (DKIM): DKIM adds a cryptographic signature to outgoing emails, verifying they weren't altered in transit. Generate and activate DKIM keys in Salesforce and add the provided CNAME records to your DNS. [Learn more about DKIM setup](https://help.salesforce.com/s/articleView?id=sf.emailadmin_setup_dkim_key.htm&type=5).
+3. Domain-based Message Authentication, Reporting, and Conformance (DMARC): DMARC builds on SPF and DKIM to define how email receivers should handle authentication failures and provides reporting. [Learn more about DMARC](https://help.salesforce.com/s/articleView?id=sf.emailadmin_dmarc.htm&type=5).
+
+## Create and Send Emails in Salesforce Marketing Cloud
+
+### Step 1: Log in and Access Email Studio
+
+1. Navigate to your [Salesforce Marketing Cloud login page](https://mc.exacttarget.com/cloud/login.html) and log in with your credentials.
+
+   <img src="{{base_path}}/assets/img/integrate/connectors/sfmc/sfmc-login.png" title="SFMC login" width="50%" alt="SFMC login"/>
+
+2. Navigate to Email Studio from the main dashboard.
+
+### Step 2: Develop Your Email Template
+
+1. Access the Email Templates Home. Within Email Studio, locate the Content section or Email Templates.
+
+   <img src="{{base_path}}/assets/img/integrate/connectors/sfmc/email-template-home.png" title="Email Template Home" width="50%" alt="Email Template Home"/>
+
+2. Click “New Email Template,” fill in the required details, and save.
+
+   <img src="{{base_path}}/assets/img/integrate/connectors/sfmc/email-template-new.png" title="New Email Template" width="100%" alt="New Email Template"/>
+
+3. Open the template details page and click "Edit in Builder." From there, you can modify the HTML, paste an existing template, or choose from their available templates. I have also added a custom HTML template, which you can find [here](https://gist.github.com/aashikam/999fc472ebc16e06be4e72f644fff55d).
```

Comment:
```suggestion
3. Open the template details page and click **Edit in Builder**. From there, you can modify the HTML, paste an existing template, or choose from their available templates.
```
Remove this phrase. We can upload this file to the attachments folder in the doc space to be downloaded. 

Download a custom HTML template [here](`insert link`).